staking/apr/compute.go

```diff
@@ -117,15 +124,7 @@ func ComputeForValidator(
 		return nil, err
 	}
 
-	if estimatedRewardPerYear.Cmp(common.Big0) == 0 {
```

Comment:
leave this one, we can have specific or other business logic for `expectedRewardPerYear`